deprecations.js

```diff
@@ -1,11 +0,0 @@
-'use strict'
```

Comment:
Please keep the file. We can comment out the two warning lines. We will want the file for future deprecations.

Reply:
@jsumners ok, I restored the file and I commented the warning name and an example for the future deprecations.